docs/MockFunctionAPI.md

```diff
@@ -594,6 +607,40 @@ test('returns correct data', () => {
 
 Types of classes, functions or objects can be passed as type argument to `jest.Mocked<Source>`. If you prefer to constrain the input type, use: `jest.MockedClass<Source>`, `jest.MockedFunction<Source>` or `jest.MockedObject<Source>`.
 
+### `jest.Replaced<Source>`
+
+The `jest.Replaced<Source>` utility type returns the `Source` type wrapped with type definitions of Jest mock property.
+
+```ts title="src/utils.ts"
```

Comment:
```suggestion
Constructs the type of a replaced property.

```ts title="src/isLocalhost.ts"
```

Reply:
There are multiple examples across the .md files that present an example of Jest API while relying on some methods from `utils.ts`... I liked that approach, as it feels generic enough so that one can focus more on the test case...

Is it okay to keep it like that?

Reply:
yeah, I think so